src/map/hash_trie_map/mod.rs

```diff
@@ -856,5 +865,60 @@ impl<'a, K, V> Iterator for Iter<'a, K, V>
 
 impl<'a, K: Eq + Hash, V> ExactSizeIterator for Iter<'a, K, V> {}
 
+#[cfg(feature = "serde")]
+impl<K, V, H> Serialize for HashTrieMap<K, V, H> where
+    K: Eq + Hash + Serialize,
+    V: Serialize,
+    H: BuildHasher + Clone + Default {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let mut state = serializer.serialize_map(Some(self.size()))?;
+        for (k, v) in self {
+            state.serialize_entry(&k, &v)?;
+        }
+        state.end()
+    }
+}
+
+#[cfg(feature = "serde")]
+impl<'de, K, V, H> Deserialize<'de> for HashTrieMap<K, V, H> where
+    K: Eq + Hash + Deserialize<'de>,
+    V: Deserialize<'de>,
+    H: BuildHasher + Clone + Default {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
+        deserializer.deserialize_map(HashTrieMapVisitor{phantom: PhantomData})
```

Comment:
spaces around `{` for consistency (also apply in other places)

Reply:
Do you mean `deserializer.deserialize_map(HashTrieMapVisitor { phantom: PhantomData } )`, or some other combination of spaces?

Reply:
> Do you mean
>
> `deserializer.deserialize_map(HashTrieMapVisitor { phantom: PhantomData } )`

Yes